src/datastore/pg-store.ts

```diff
@@ -101,6 +101,8 @@ import { parseBlockParam } from '../api/routes/v2/schemas';
 
 export const MIGRATIONS_DIR = path.join(REPO_DIR, 'migrations');
 
+const TRGM_SIMILARITY_THRESHOLD = 0.3;
```

Comment:
Why `0.3`? Should we make this an ENV variable to be able to tweak it on our deployments?

Reply:
It's actually the default similarity factor (thought defining it explicitly would make it clearer and easier to change if needed), I don't see it changing frequently unless we decide to make the search term more forgiving/strict in the future (based on user feedback, etc.).